### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by jfrankl**
image

### CODE REVIEW

File: src/nyc_trees/apps/event/templates/event/event_detail.html [hunk 1]
```diff
@@ -15,108 +15,149 @@
...
-            </p>
+        <div class="row">
+            <div class="col-xs-4">
+                <div class="calendar">
+                    <div class="calendar-heading">
+                        {{ event.begins_at|date:"M" }}
+                    </div>
+                    <div class="calendar-body">
+                        {{ event.begins_at|date:"d" }}
+                    </div>
```
Comment (kdeloach) [inline src/nyc_trees/apps/event/templates/event/event_detail.html]: The calender block is cool.

**Comment by kdeloach** [comment]
The RSVP button no longer works for non-admins.

**Comment by kdeloach** [comment]
The container div around the RSVP button needs `id="rsvp-section"`.

**Comment by kdeloach** [comment]
+1

**Comment by kdeloach** [comment]
Looks good.

**Comment by jfrankl** [comment]
Good catch on the id="rsvp-section"!

**Pull request merged**